modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -974,6 +974,38 @@ void ONNXImporter::populateNet(Net dstNet)
             }
             replaceLayerParam(layerParams, "mode", "interpolation");
         }
+        else if (layer_type == "FusedUpsample")
+        {
+            if (node_proto.input_size() != 6)
+                CV_Error(Error::StsNotImplemented,
+                         "Expected input, indices1, B1, indices2, B2, inpIndex");
+
+            layerParams.type = "Resize";
+
+            cv::Mat indices1 = getBlob(node_proto, constBlobs, 1);
+            cv::Mat indices2 =  getBlob(node_proto, constBlobs, 3);
+            cv::Mat B1 = getBlob(node_proto, constBlobs, 2);
+            cv::Mat B2 =  getBlob(node_proto, constBlobs, 4);
+            cv::Mat inpIndex = getBlob(node_proto, constBlobs, 5);
+
+            CV_Assert(indices1.total() == 1);
+            CV_Assert(indices2.total() == 1);
+            CV_Assert(B1.total() == 1);
+            CV_Assert(B2.total() == 1);
+            CV_Assert(inpIndex.total() == 2 && (int)inpIndex.at<float>(0) == 1 && (int)inpIndex.at<float>(1) == 1);
+
+            if(indices1.at<int>(0) == 2 && indices2.at<int>(0) == 3)
+            {
+                layerParams.set("zoom_factor_y", B1.at<int>(0));
+                layerParams.set("zoom_factor_x", B2.at<int>(0));
+            }
+            else if(indices2.at<int>(0) == 2 && indices1.at<int>(0) == 3)
+            {
+                layerParams.set("zoom_factor_y", B2.at<int>(0));
+                layerParams.set("zoom_factor_x", B1.at<int>(0));
+            }
+            replaceLayerParam(layerParams, "mode", "interpolation");
+        }
```

Comment:
Can we make a fused layer to match existing implementation?

https://github.com/opencv/opencv/blob/221564d12c6183f2d15515e38a7d2d0f0c4bdc12/modules/dnn/src/onnx/onnx_importer.cpp#L958

Reply:
`Scales` are attribute in `Upsample` layer. 
But inputs `B` of `mul `layers in unfused subgraph. So either we need to extract them in graph_simplifier.cpp or in onnx_importer.cpp (current implementation).

Reply:
Yeah, I mean we can extract the exact values from them at finalize step and put to scales